system_baseline/views/admin.py

```diff
@@ -15,7 +15,7 @@ def status():
     dt = date.today()
 
     total_baselines = SystemBaseline.query.with_entities(SystemBaseline.id).distinct().count()
-    customer_count = SystemBaseline.query.with_entities(SystemBaseline.account).distinct().count()
+    customer_count = SystemBaseline.query.with_entities(SystemBaseline.org_id).distinct().count()
```

Comment:
do we need an if/else here or are we sure all will be populated with org_id?

Reply:
This is part of reporting, answered above.